get_gcs_test.go

```diff
@@ -108,13 +108,13 @@ func TestGCSGetter_GetGenerationFile(t *testing.T) {
 	if err != nil {
 		t.Fatalf("err: %s", err)
 	}
-	if string(content) != "a" {
+	if string(content) != "a\n" {
```

Comment:
Is there a reason we're adding the newlines to the test data?

Reply:
Nope, I created the file using `echo a > file.txt` and it added a `\n`, and I just felt like this was an okay change.

Reply:
That's fair. Trailing newline seems like a good practice - I just didn't know if there's anything more to it 😄

Reply:
Thanks for reviewing !